### PR TITLE
Update icon for Shop Pay rebranding

### PR DESCRIPTION
**Pull request by magalhini**
On Debut:

image

The source for the guidelines is internal on Shopify Pay, coming from our official style guide:

image

To note that we're still planning on keeping the name `shopify_pay` as the source of the payment as not to change any developer facing keys for now.

** Checklist **
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [ ] I have provided a link to the brand icon’s brand guidelines whenever possible.

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

File: app/assets/images/payment_icons/shopify_pay.svg [hunk 1]
```diff
@@ -1 +1 @@
...
...692 0 2.303.783 2.303 2.276v3.172a.13.13 0 01-.132.129h-1.03a.13.13 0 01-.13-.13v-.236a.096.096 0 00-.061-.091.099.099 0 00-.107.022c-.31.334-.808.575-1.607.575-1.175 0-1.95-.607-1.95-1.657zm3.735-.687v-.246l-1.339.07c-.705.036-1.115.326-1.115.816 0 .444.376.69 1.034.69.893 0 1.42-.48 1.42-1.33zM27.755 17.417v-.919a.131.131 0 01.157-.127c.158.029.318.044.479.044a1.229 1.229 0 001.245-.876l.067-.211a.133.133 0 000-.088l-2.145-5.471a.13.13 0 01.122-.18h1.04a.132.132 0 01.123.085l1.456 3.859a.131.131 0 00.125.088.133.133 0 00.125-.088l1.265-3.848a.13.13 0 01.126-.09h1.076a.134.134 0 01.132.116.133.133 0 01-.008.063l-2.295 6.076c-.528 1.413-1.433 1.773-2.43 1.773a1.96 1.96 0 01-.561-.066.132.132 0 01-.1-.14zM8.57 6.4a5.363 5.363 0 00-3.683 1.427.231.231 0 00-.029.31l.618.839a.236.236 0 00.276.084.237.237 0 00.086-.056 3.823 3.823 0 012.738-1.11c2.12 0 3.227 1.584 3.227 3.15 0 1.7-1.163 2.898-2.835 2.921-1.292 0-2.266-.85-2.266-1.974a1.908 1.908 0 01.713-1.48.231.231 0 00.033-.324l-.65-.815a.236.236 0 00-.339-.034c-.399.32-.72.724-.942 1.183a3.39 3.39 0 00-.337 1.47c0 1.935 1.655 3.452 3.775 3.464h.03c2.517-.032 4.337-1.884 4.337-4.415 0-2.247-1.667-4.64-4.752-4.64z" fill="#fff"/></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/shopify_pay.svg]: The border radius looks different from other payment icon. Could we adjust it?

[video example](https://screenshot.click/30-25-bj2rl-h283o.mp4)

Comment (magalhini) [reply app/assets/images/payment_icons/shopify_pay.svg]: Excellent point, thank you for pointing that out! I've asked the design team to give it the recommended 2px border from the guidelines 👍

**Comment by magalhini** [comment]
Hey @tauthomas01 👋 we're planning on shipping the rebranding very very soon (early next week). Is there any plans for a release for this? Thanks!

**Comment by tauthomas01** [comment]
Hello @magalhini , I can release next Monday. How does that sound?

**Comment by magalhini** [comment]
@tauthomas01 that would be perfect, please keep us posted so we can make the change and have a smooth rebranding 😄 we'll keep an eye out on Monday. Thank you!

**Comment by magalhini** [comment]
The Shop Pay rebranding has launched today @tauthomas01 🎉 please let us know if there's anything we can do to ensure this goes in soon, as it's the only remaining asset to complete the SHOP rebranding 😄 we'd be happy to help!

**Comment by tauthomas01** [comment]
@magalhini Okay, I will merge this and create a release tomorrow morning. If it's too late, it will be available Monday at the latest.

**Pull request merged**